rosidl_generator_c/test/test_interfaces.c

```diff
@@ -295,6 +310,10 @@ int test_strings(void)
   EXPECT_EQ(true, res);
   EXPECT_EQ(0, strcmp(strings->empty_string.data, TEST_STRING));
   EXPECT_EQ(0, strcmp(strings->def_string.data, "Hello world!"));
+  EXPECT_EQ(0, strcmp(strings->def_string2.data, "Hello'world!"));
+  EXPECT_EQ(0, strcmp(strings->def_string3.data, "Hello\"world!"));
+  EXPECT_EQ(0, strcmp(strings->def_string4.data, "Hello\'world!"));
```

Comment:
Based on the message definition:

> string def_string4 'Hello\'world!'

Why is there a backslash in the C++ string literal? Should it be just `"Hello'world!"`?

Same for [def_various_quotes.data[1].data](https://github.com/ros2/rosidl/pull/197/files#diff-be6a8e8a931ac60ef4902d04a32978b0R430).

Reply:
hmmm good question. I guess as the escaping is not necessary in C because we use double quotes it should get "unescaped" by the C generator when it gets it from the parser

Reply:
The extra backslash should be removed then.

Reply:
Done in efe98ea02a0a9a49f4173ab5c80ca9b9fd5556a8